docs/built-in-bpa-rules.md

```diff
@@ -6,116 +6,116 @@ Information about the PowerShell-based rules included by our integration with [P
 
 ## JSON-Based Rules:
 
-### TA-000001: Diagnostic logs in App Services should be enabled
+### TA-000001: Diagnostic logs in app services should be enabled
```

Comment:
```suggestion
### TA-000001: Diagnostic logs in App Services should be enabled
```

Reply:
Why do you think it should be upper case?

Reply:
Because it is called `Azure App Services` and I looked at the way it was referred to in [this document ](https://azure.microsoft.com/en-us/products/app-service/#overview) and they used upper. I also looked at the other ones in documents to see how they were referred to and they used both upper and lower casing.

Reply:
Okay, I can change it to upper case everywhere, we were also using both casings

Reply:
I meant the other ones are ok. Just change this one

Reply:
* Just change it for App Services.